packages/xod-project/src/custom-types/index.js

```diff
@@ -0,0 +1,45 @@
+import * as R from 'ramda';
+import { isAmong } from 'xod-func-tools';
+
+import Color from './color';
+
+// =============================================================================
+//
+// Add custom type specifications in BINDABLE_CUSTOM_TYPES_SPECS
+// Key — an alias for easy access by a Developer
+// Value — specification object
+//
+// =============================================================================
+
+const BINDABLE_CUSTOM_TYPES_SPECS = {
+  COLOR: Color,
```

Comment:
I see no reason to keep `Color` (and subsequent types) in own trivial modules. Suggest simplification: embed the specs right here.

Reply:
👌